services/neo4j/init/neo4j-queries.cql

```diff
@@ -1,4 +1,5 @@
 CREATE CONSTRAINT ON (`config`:`config`) ASSERT `config`.`entityId` IS UNIQUE;
+CREATE CONSTRAINT ON (`kilda_configuration`:`kilda_configuration`) ASSERT `kilda_configuration`.`entityId` IS UNIQUE;
```

Comment:
Do we need a migration script to update already deployed Kilda instance?

Reply:
added